docs/schemas/ros2_type_description.schema.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
@james-rms I am bit confused with json file. As far as I recall in  one of your original gist comments you mention that you have changed format to yaml.